hass_nabucasa/voice.py

```diff
@@ -48,7 +48,7 @@ class AudioOutput(str, Enum):
     RAW = "raw"
 
 
-# The first entry for each language is the default voice
```

Comment:
This (in some languages) will "change default voice." IMO if a default voice wants to be set, that is logic that should live in Home Assistant and not here anyway.

Reply:
We cant do a breaking change on default voices. So if you change it here, defaults should first be migrated to Home Assistant before this upgrade is done

Reply:
And if this was a PR in core i would agree, but its not.